central/globaldb/postgres.go

```diff
@@ -104,6 +98,52 @@ func GetPostgres() *pgxpool.Pool {
 	return postgresDB
 }
 
+// GetPostgresConfig - gets the configuration used to connect to Postgres
+func GetPostgresConfig() (map[string]string, *pgxpool.Config, error) {
+	centralConfig := config.GetConfig()
+	password, err := os.ReadFile(dbPasswordFile)
+	if err != nil {
+		log.Fatalf("pgsql: could not load password file %q: %v", dbPasswordFile, err)
+		return nil, nil, err
+	}
+	source := fmt.Sprintf("%s password=%s", centralConfig.CentralDB.Source, password)
+
+	config, err := pgxpool.ParseConfig(source)
+	if err != nil {
+		log.Fatalf("Could not parse postgres config: %v", err)
+		return nil, nil, err
+	}
+
+	sourceMap, err := ParseSource(source)
```

Comment:
What fields are we making use of that require us to also parse the source instead of using fields that are within the config?

Reply:
@connorgorman The SSL mode and cert location.  I chose to set that stuff in an env var and there was no real easy way to get it out of the connection.
See `central/globaldb/v2backuprestore/common/postgres.go`. 
There were equally clumsy ways around it where I could pass the source in a quoted strings, but that made feeding the options to cmd more difficult.  At least it did for me.